server/middlewares/authorizeUser.js

```diff
@@ -0,0 +1,20 @@
+import models from '../database/models';
+import responseMessage from '../helpers';
+
+const { UserRole, Role } = models;
+
+export default async (request, response, next, roleArray) => {
+  try {
+    const { id } = request.userData;
```

Comment:
What is userData?

Reply:
Userdata is attached to the request from in the verify token middleware. So I am getting the id from there.